model.py

```diff
@@ -577,7 +569,7 @@ class Place(Base):
     # The geography of the place itself. It is stored internally as a
     # geometry, which means we have to cast to Geography when doing
     # calculations.
-    geometry = Column(Geometry(srid=4326))
+    geometry = Column(Geometry(srid=4326), nullable=True)
```

Comment:
I also find this weird because I thought I set nullable=True in one of my branches.